apps/synapse-portal-framework/src/README.md

```diff
@@ -72,3 +72,11 @@ The route resolver handles taking a URL and matching it the config. Read more in
 
 The detail pages ([e.g.](https://staging.adknowledgeportal.synapse.org/Explore/Studies/DetailsPage?Study=syn5550404])) give
 a deeper dive into a particular portal section. Read more in the [component](portal-components/DetailsPage/DetailsPage.tsx).
+
+# How to Log In to a Portal Running on Localhost
+
+To log in to the portal running on localhost, follow these steps:
+
+1. Start the SageAccountWeb application locally by navigating to the app directory and running pnpm start (located in apps/SageAccountWeb).
+2. Once the application is running, log in to the local Accounts site.
+3. After successfully logging in, refresh the portal page to complete the process.
```

Comment:
We can probably add some orchestration with Nx so if you run `nx run myportal:start` we can automatically run `nx run SageAccountWeb:start`. We might want to give SageAccountWeb a unique port so we can reliably redirect to it.